### DISCUSSION TITLE
First beta version of Jamulus 3.8.0 has been released

### DISCUSSION THREAD

**Discussion by softins** [jamulussoftware/jamulus#1603] [Announcements]
**THIS IS A BETA VERSION! Expect the possibility of bugs, and please tell us about them.**

We have just released version **3.8.0beta1** of Jamulus, and would be happy for you to download, compile and test out this version.

You can view the Release Notes and download versions for Windows, Mac, Linux (Debian/Ubuntu) and Android (still experimental) at https://github.com/jamulussoftware/jamulus/releases/tag/r3_8_0beta1. You can also download the source code as a zip or tarball for compiling, or you can check out the code using `git`.

If you find any issues with it, please raise an Issue with as many details as possible.

**Comment by videobuff** [comment]
Installed two servers now with 3.8 from source on a Centos and Debian server. First impression is more stable. FYI, the about still shows the 3.7 info.
![Schermafbeelding 2021-05-05 om 16 19 05](https://user-images.githubusercontent.com/3132953/117156588-23be5600-adbe-11eb-80c9-1be3f78db69e.jpg)
![Schermafbeelding 2021-05-05 om 16 18 28](https://user-images.githubusercontent.com/3132953/117156594-2620b000-adbe-11eb-81f3-044a2763e197.jpg)

**Reply by softins** [reply]
The About window shows the version of client that you are running on your Mac, not the server version.

It looks like you are running on Big Sur, is that correct?

**Comment by videobuff** [comment]
Tony
🙈How stupid one can be🙈 Yes the client is running on big sur. 

Verstuurd vanaf mijn telefoon
Groeten
Erik

**Reply by videobuff** [reply]
Well, the client on my Mac with Big Sur is also sorted out. Tonight testing with both client and server at 3.8 beta.

![Schermafbeelding 2021-05-05 om 18 08 45](https://user-images.githubusercontent.com/3132953/117173422-0ba20300-adcd-11eb-8d7f-1bc351fe7081.jpg)

**Comment by 38github** [comment]
Is there any video demonstrating the new delay panning feature? It "sounds" interesting but I do not want to overwrite my stable server installation until the final release is out. Maybe some one has setup a 3.8b server that we/I can try on?

**Reply by hoffie** [reply]
> Maybe some one has setup a 3.8b server that we/I can try on?

I think @pljones mentioned that his jamulus.drealm.info (listed on Any Genre 2) has delay panning enabled. That's not a specific test server though, so the general rules of not disturbing others by testing apply. It's currently unused though.

Everyone who tried delay panning reported back that they would no longer want to miss it (me included). :)

**Reply by dcorson-ticino-com** [reply]
My server is running 3_8_0beta1 with delay panning on.  I think it is great.
You can try it on the directory Classical, Strings Server

**Comment by 38github** [comment]
Is this this new feature Delay Panning applied when I change the panning of other users or is it only applied when I change my own panning in Settings?

Is there any audio or video file that demonstrates the difference?

**Reply by dcorson-ticino-com** [reply]
Hi 38,
The delay panning is a server function and is for all clients on the server.  As a client you can not turn it on or off.
You can try it on my server mentioned in a post above.

Note that the Balance or Pan on the Settings advanced tab sets the relative volume of your two inputs, it does not really pan.
